contracts/staking/validatorShare/ValidatorShareFactory.sol

```diff
@@ -7,16 +7,14 @@ contract ValidatorShareFactory {
     - factory to create new validatorShare contracts
    */
 
-    function create(
-        uint256 validatorId,
-        address loggerAddress,
-        address governance
-    ) public returns (address) {
+    function create(uint256 validatorId, address loggerAddress)
+        public
+        returns (address)
+    {
         ValidatorShare validatorShare = new ValidatorShare(
             validatorId,
             loggerAddress,
-            msg.sender,
-            governance
+            msg.sender
         );
         validatorShare.transferOwnership(msg.sender);
```

Comment:
Instead of this, we could call `_transferOwnership` in `validatorShare` constructor since we are passing the `stakeManager` address now as a param in the constructor.

Reply:
ah yes fixed